Sources/SwiftDocC/Semantics/Metadata/PageImage.swift

```diff
@@ -11,19 +11,26 @@
 import Foundation
 import Markdown
 
+/// A directive to adjust the image associated with the page.
+///
+/// This directive can be used to control the image used both when rendering an icon representing the page such as
+/// in the navigator sidebar, and when rendering a card representing a page, e.g. in a  Topics section styled as a grid.
 public final class PageImage: Semantic, AutomaticDirectiveConvertible {
     public let originalMarkup: BlockDirective
     
+    /// The purpose for which to use this iamge.
     @DirectiveArgumentWrapped
     public private(set) var purpose: Purpose
     
+    /// The path at which to find the image resource.
     @DirectiveArgumentWrapped(
         parseArgument: { bundle, argumentValue in
             ResourceReference(bundleIdentifier: bundle.identifier, path: argumentValue)
         }
     )
     public private(set) var source: ResourceReference
     
+    /// Alt text to provide for users with accesibility needs.
```

Comment:
I prefer the phrasing from [the @Image documentation](https://www.swift.org/documentation/docc/image)

```suggestion
    /// Alternative text that describes the image to screen readers.
```